File: extensions/adloader-core/src/main/java/com/mxplay/adloader/AdsBehaviourOffline.java
package com.mxplay.adloader;

public class AdsBehaviourOffline extends AdsBehaviourDefault {
    public AdsBehaviourOffline(int vastTimeOutInMs) {
        super(vastTimeOutInMs);
    }

    @Override
    public String getTrackerName() {
        return VideoAdsTracker.OFFLINE_AD_LOADER;
    }
}

File: extensions/adloader-core/src/main/java/com/mxplay/adloader/AdsBehaviourOffline.kt
package com.mxplay.adloader

class AdsBehaviourOffline(vastTimeOutInMs: Int, videoAdsTracker: VideoAdsTracker? = null, debug : Boolean = false) : AdsBehaviourDefault(vastTimeOutInMs, videoAdsTracker, debug) {
    override val trackerName: String
        get() = VideoAdsTracker.OFFLINE_AD_LOADER
}